rules.yml

```
@@ -5,7 +5,7 @@ is_less_than: '^[ ]\b(is_less_than)\b: ([\s\d]+)$'
 is_equal_to: '^[ ]\b(is_equal_to)\b: ([\s\d]+)$'
 is_different_to: '^[ ]\b(is_different_to)\b: ([\s\d]+)$'
 is_globally_unique: '^[ ]\b(is_globally_unique)\b'
-refers_to: '^[ ]\b(refers_to)\b'
+refers_to: '^[ ]\b(refers_to)\b: (?:[^,\[\]\s]+|\[\s*(?:[^,\[\]\s]+\s*,\s*)*[^,\[\]\s]+\s*\])$'
```

Maybe make argument group capturing, as the other entries do - even though that is currently not used anywhere, it at least keeps it consistent:

```suggestion
refers_to: '^[ ]\b(refers_to)\b: ([^,\[\]\s]+|\[\s*(?:[^,\[\]\s]+\s*,\s*)*[^,\[\]\s]+\s*\])$'
```